esx_service/cli/vmdkops_admin.py

```diff
@@ -172,83 +175,186 @@ def commands():
+                                    'help': 'Tenant name',
+                                    'required': True
+                                },
+                                '--datastore': {
+                                    'help': "Datastore name",
+                                    'required': True
+                                },
+                                '--add-rights': {
+                                    'help': 'Datastore access Permissions granted',
+                                    'choices': ['create', 'delete', 'mount', 'all'],
+                                    'metavar': 'create,delete,mount'
+                                }, 
+                                '--rm-rights': {
+                                    'help': 'Datastore access Permissions granted',
+                                    'choices': ['create', 'delete', 'mount', 'all'],
+                                    'metavar': 'create,delete,mount'
+                                },
+                                '--volume-maxsize': {
+                                    'help': 'Maximum size of the volume that can be created',
+                                    'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
+                                },
+                                '--volume-maxcount': {
+                                    'help': 
+                                    'Maximum number of volumes to create on the datastore for this tenant'
+                                },
+                                '--volume-totalsize': {
+                                    'help': 
+                                    'Maximum total size of all volume that can be created on the datastore for this tenant',
+                                    'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
+                                }
+                            }
+                        },
+                        'rm': {
+                            'func': tenant_access_rm,
+                            'help': "Remove all access to a datastore for a tenant",
+                            'args': {
+                                '--name': {
+                                    'help': 'The name of the tenant',
+                                    'required': True
+                                },
+                                '--datastore': {
+                                    'help': "Datstore which access is controlled",
+                                    'required': True
+                                }
+                            }
+                        },
+                        'ls': {
+                            'func': tenant_access_ls,
+                            'help': 'List all access info for a tenant',
+                            'args': {
+                                '--name': {
+                                    'help': 'The name of the tenant',
+                                    'required': True
```

Comment:
Again, list for named tenants or make required as "False" and when not provided list access for all tenants.